src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -90,7 +90,7 @@
     private String krb5LoginConfFile;
     private String krb5ConfFile;
     private boolean serviceAsNamespace;
-    private HDFSBackend persistenceBackend;
+    private HDFSBackendImpl persistenceBackend;
```

Comment:
Not to be solved in this PR, but I noticed that the naming criteria used for this class is not the same as the rest... I mean it should be HdfsBackendImpl instead of HDFSBackendImple, right? Just in case you consider it appropriate to create an issue about it ;)